webapp/i18n/en.json

```diff
@@ -1,3 +1,26 @@
 {
-  "button.channel.menu": "Disable Voice"
-}
+  "/+63eJ": "Set up RTCD Services",
```

Comment:
This automation is great but could losing the human readable ids become a problem?

Reply:
Discussed this internally. Same approach has been used in Playbooks and it's working well.